lib/SPIRV/SPIRVReader.cpp

```diff
@@ -696,7 +698,8 @@ bool SPIRVToLLVM::isDirectlyTranslatedToOCL(Op OpCode) const {
     // clang-consistent format in SPIRVToOCL pass.
     return !(isAtomicOpCode(OpCode) || isGroupOpCode(OpCode) ||
              isGroupNonUniformOpcode(OpCode) || isPipeOpCode(OpCode) ||
-             isMediaBlockINTELOpcode(OpCode));
+             isMediaBlockINTELOpcode(OpCode) || OpCode == OpGroupAsyncCopy ||
```

Comment:
Does it make sense to create it to `isGroupEventAlgorithm` function?

Reply:
It would make sense if we merge `SPIRVToOCL::visitCallAsyncWorkGroupCopy` and `SPIRVToOCL::visitCallGroupWaitEvents` (I added them in this patch) so we can use it in `SPIRVToOCL` pass, because I believe we will remove `isDirectlyTranslatedToOCL` function one day.